relayer-cli/src/commands/listen.rs

```diff
@@ -51,29 +91,42 @@ impl Runnable for ListenCmd {
 }
 
 /// Listen to events
-pub fn listen(config: &ChainConfig, events: &[EventType]) -> Result<(), BoxError> {
+pub fn listen(config: &ChainConfig, filters: &[EventFilter]) -> Result<(), BoxError> {
     println!(
         "[info] Listening for events `{}` on '{}'...",
-        events.iter().format(", "),
+        filters.iter().format(", "),
         config.id
     );
 
     let rt = Arc::new(TokioRuntime::new()?);
-    let queries = events.iter().cloned().map(Query::from).collect();
-    let (event_monitor, rx) = subscribe(&config, queries, rt)?;
+    let (event_monitor, rx) = subscribe(&config, rt)?;
 
     thread::spawn(|| event_monitor.run());
 
     while let Ok(event_batch) = rx.recv() {
-        println!("{:#?}", event_batch);
+        match event_batch {
+            Ok(batch) => {
+                println!("- Event batch at height {}", batch.height);
```

Comment:
Can we print this only if events are filtered below? Otherwise we end up with something like this filling the output:
```
- Event batch at height 1-55

- Event batch at height 1-56

- Event batch at height 1-57

- Event batch at height 1-58

- Event batch at height 1-59

```

Reply:
Fixed in e6693ef9